website/src/site/markdown/index.md

```diff
@@ -16,36 +16,35 @@ KIND, either express or implied.  See the License for the
-
-The official home for the REEF (and Tang and Wake) source code is at the Apache Software Foundation. You can check out the current code via:
-
-    $ git clone git://git.apache.org/reef.git
-
-or directly access its GitHub page [here](https://github.com/apache/reef).
-
-Detailed information about REEF and using it can be found in the [FAQ](faq.html) and the [Tutorial](https://cwiki.apache.org/confluence/display/REEF/Tutorials).
-
-If you wish to contribute, start at the [Contributing](https://cwiki.apache.org/confluence/display/REEF/Contributing) tutorial or the [Committer Guide](https://cwiki.apache.org/confluence/display/REEF/Committer+Guide)!
-
-###Further questions?
-
-Please visit our [Frequently Asked Questions](faq.html) page or use our [Mailing List](mailing-list.html) to send us a question!
+Apache REEF&trade; - a `stdlib` for Big Data
+=============================================
+Apache REEF&trade; (Retainable Evaluator Execution Framework) is a  library for developing portable applications for cluster resource managers such as [Apache Hadoop&trade; YARN][YARN] or [Apache Mesos&trade;][Mesos]. Apache REEF drastically simplifies development of those resource managers through the following features:
+
+Features
+--------
+ * **Centralized Control Flow:** Apache REEF turns the chaos of a distributed application into events in a single machine, the Job Driver. Events include container allocation, Task launch, completion and failure. For failures, Apache REEF makes every effort of making the actual `Exception` thrown by the Task available to the Driver.
+ * **Task runtime:** Apache REEF provides a Task runtime called Evaluator. Evaluators are instantiated in every container of a REEF application. They can be reused for subsequent tasks as well as support to handover data between such tasks.
+ * **Support for multiple resource managers:** Apache REEF applications are portable between any supported resource manager with minimal effort. Further, new resource managers are easy to support in REEF. 
+ * **.NET and Java API:** Apache REEF is the only API to write YARN or Mesos applications in .NET. Further, a single REEF application is free to mix and match Tasks written for .NET or Java.
+ * **Plugins:** Apache REEF allows for plugins (called "Services") to augment its feature set without adding bloat to the core. REEF includes many Services, such as a name-based communications between Tasks, MPI-inspired group communications (`Broadcast`, `Reduce`, `Gather`, ...) and data ingress.   
+
+REEF @ Apache Big Data 2016
+---------------------------
+
+<iframe src="//www.slideshare.net/slideshow/embed_code/key/1HgrkEUWqqD2x1" width="595" height="485" frameborder="0" marginwidth="0" marginheight="0" scrolling="no" style="border:1px solid #CCC; border-width:1px; margin-bottom:5px; max-width: 100%;"> </iframe>
+
+Latest Release
+--------------
+The latest release of Apache REEF is 0.14.0. Please go to the [downloads] page to find out more.
```

Comment:
Please update `update_website.py` script so that it updates this mention of version also.